core/src/main/kotlin/api/RepositoriesRoute.kt

```diff
@@ -26,7 +26,6 @@ import io.github.smiley4.ktorswaggerui.dsl.routing.post
 import io.github.smiley4.ktorswaggerui.dsl.routing.put
 
 import io.ktor.http.HttpStatusCode
-import io.ktor.server.application.call
```

Comment:
Nit: This is not related to the change and therefore should be in a separate commit.